src/pages/index.tsx

```diff
@@ -93,14 +110,18 @@ export default function Home() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Header />
-      <main className="flex min-h-screen flex-col pt-36 sm:flex-row sm:pt-24">
-        <Filters
-          filters={{ search, countries, positions, tags, withAudio }}
-          onChange={onChangeFilter}
-          onClear={onClearFilter}
-        />
-        <div className="flex-1">
+      <Header isMenuOpen={isMenuOpen} toggleMenu={toggleMenu} />
+      <main className="flex flex-col sm:flex-row sm:pt-6 lg:pt-36">
+        <div className={`sm:w-80 ${isSmSize && !isMenuOpen ? 'hidden' : 'static'} ${isMenuOpen ? ' z-40 mt-10 ' : ''}`}>
```

Comment:
Why do you need the `sm` size in an state? You can create the proper class and merge it with `sm:`
Also `static` is not the opposite of `hidden`. I don't get this. 
Another point: `${isMenuOpen ? ' z-40 mt-10 ' : ''}`}` can be `${isMenuOpen && ' z-40 mt-10'}`